docker/Dockerfile

```diff
@@ -7,15 +7,12 @@ ENV TZ="America/New_York"
 RUN apk add --update python3 py-pip git tzdata && \
     rm -rf /var/cache/apk/*
 
-RUN mkdir -p /opt/ha-wyzesense && \
-    git clone -b "0.0.7" https://github.com/kevinvincent/ha-wyzesense /opt/ha-wyzesense
-
 RUN mkdir -p /opt/wyzesense2mqtt && \
-    git clone https://github.com/raetha/wyzesense2mqtt.git /opt/wyzesense2mqtt && \
+    git clone -b "v0.5" https://github.com/raetha/wyzesense2mqtt.git /opt/wyzesense2mqtt && \
```

Comment:
Good reminder, thanks. Though still trying to understand best practice for version dependencies. Some places seem to imply you should always hard set the version to prevent any unintended breaking changes.

Reply:
I don't know, I don't like the idea of stuff in `master` pointing to another branch.

Reply:
Actually that same format is used to point to tags, and this branch will die once merged, but the tag/release will stay. That said, I'm probably going to point it to master anyway. Just been seeing a lot of stuff that is nitpicky about having you specify explicit versions of modules, libraries, etc. Something about avoiding breaking changes....blah blah blah.

Reply:
Explicit versions of module and library _dependencies_, aka "other people's code" not your application (/module/library/script/whatever).